src/ducks/recurrence/RecurrencePage.jsx

```diff
@@ -321,10 +321,16 @@ const BundleInfo = ({ bundle }) => {
               <Img className="u-flex" style={imgLineHeightStyle}>
                 <ActionMenuHelper
                   opener={
-                    <Button extension="narrow" theme="secondary">
-                      {t(`Recurrence.status.${getStatus(bundle)}`)}
-                      <Icon className="u-ml-half" icon={BottomIcon} />
-                    </Button>
+                    <Button
+                      extension="narrow"
+                      theme="secondary"
+                      label={
+                        <>
+                          {t(`Recurrence.status.${getStatus(bundle)}`)}
+                          <Icon className="u-ml-half" icon={BottomIcon} />
```

Comment:
nit: You can't use `icon + label` or `label + extraRight` because I don't think that "BottomIcon" something to do with the "label"

Reply:
I had forgotten about extraRight, thanks :)